src/dictionaries/assets/bsc-rsk-testnet-assets.ts

```diff
@@ -29,6 +33,25 @@ export const bscRskTesnetAssets = [
       Asset.SOV,
     ),
   ),
+  new AssetDetails(
+    Asset.BTCS,
+    'BTCS',
+    'BTCS',
+    rbtcIcon,
+    18,
+    '0x0ed2a1edde92b25448db95e5aa9fe9e9bc0193bf',
```

Comment:
Shouldn't this be the latest testnet BTCS token address?
```suggestion
    '0xb68faC988f5aD33044f998F363a1ECe08FD047D6',
```